src/Plugin.php

```diff
@@ -15,6 +15,8 @@ class Plugin implements
     \Composer\EventDispatcher\EventSubscriberInterface,
     \Composer\Plugin\Capable
 {
+    const COMPOSER_PACKAGE = 'vaimo/composer-patches';
```

Comment:
I'm not a huge fan of introducing random constants like that, is it possible to get this information out of composer API in a backward compatible way?

Reply:
Does not seem possible, because the [`addPlugin` method](https://github.com/composer/composer/blob/7ff288f08187f5c63001f9ad07cde948faaf644e/src/Composer/Plugin/PluginManager.php#L388) does not pass anything about the `$sourcePackage` to the plugin's activate method.

I suggest just leaving this as is.